src/command_modules/azure-cli-iotcentral/azure/cli/command_modules/iotcentral/_help.py

```diff
@@ -22,13 +22,13 @@
     short-summary: Create an IoT Central application.
     long-summary: For an introduction to IoT Central, see https://docs.microsoft.com/en-us/azure/iot-central/
     examples:
-        - name: Create an IoT Central application with the free pricing tier F1, in the region of the resource group.
+        - name: Create an IoT Central application in the standard pricing tier S1, in the region of the resource group.
           text: >
-            az iotcentral app create --resource-group MyResourceGroup --name MyApp --subdomain myapp
-        - name: Create an IoT Central application with the standard pricing tier S1 in the 'westus' region.
+            az iotcentral app create --resource-group MyResourceGroup --name my-app-resource-and-display-name --subdomain my-app-subdomain
+        - name: Create an IoT Central application with the standard pricing tier S1 in the 'westus' region, with a custom display name, based on the iotc-default template.
           text: >
-            az iotcentral app create --resource-group MyResourceGroup --name MyApp --sku S1 --location westus
-            --subdomain myapp
+            az iotcentral app create --resource-group MyResourceGroup --name my-app-resource-name --sku S1 --location westus
+            --subdomain my-app-subdomain --template iotc-default@1.0.0 --display-name 'My Custom Display Name'
```

Comment:
Where you able to successfully create the 3 different templates we have on the UI using this command?

Reply:
yes